src/test/java/io/airlift/units/TestDurationValidator.java

```diff
@@ -33,7 +33,12 @@
 
 public class TestDurationValidator
 {
-    private static final Validator VALIDATOR = Validation.byProvider(ApacheValidationProvider.class).configure().buildValidatorFactory().getValidator();
+    private static final Validator VALIDATOR = Validation.byProvider(HibernateValidator.class)
+            .configure()
+            .ignoreXmlConfiguration()
```

Comment:
why `ignoreXmlConfiguration` added?

Reply:
So that classpath is not scanned (security)

Reply:
add a code comment